homeassistant/components/sensibo/climate.py

```diff
@@ -34,12 +37,16 @@
 ATTR_MINUTES = "minutes"
 SERVICE_ENABLE_PURE_BOOST = "enable_pure_boost"
 SERVICE_DISABLE_PURE_BOOST = "disable_pure_boost"
+SERVICE_FULL_AC_STATE = "full_ac_state"
```

Comment:
Should we call this FULL_STATE or FULL_HVAC_STATE ?

Reply:
Probably better. Sensibo calls it acState probably as it's comes from ac's but can obviously also be for other climate devices. I will use `FULL_STATE` as name